.gitattributes

```diff
@@ -0,0 +1,2 @@
+lib/optimizer/resources/local_fallback/* linguist-generated=true
```

Comment:
Yes, that's right. Placing it into `lib/optimizer` will work, [ref](https://git-scm.com/book/en/v2/Customizing-Git-Git-Attributes):

![image](https://user-images.githubusercontent.com/134745/81218875-c4075200-8f93-11ea-8e59-39051e2c5bc8.png)

Reply:
As long as GitHub recognizes this, as [their docs](https://help.github.com/en/github/administering-a-repository/customizing-how-changed-files-appear-on-github) only mention the repo root.

Reply:
I assumed it would only work in the project root, but I can try moving it around to see what happens.

Reply:
No, it doesn't work when the `.gitattributes` file is in a subfolder. I'll revert that change again and leave it in the root for now. This will need to be adapted once we extract the libraries into separate repositories.

![Image 2020-05-07 at 8 29 08 AM](https://user-images.githubusercontent.com/83631/81261821-06389e00-903d-11ea-9343-706e4bed2a86.png)